plugins/movetoarchive.koplugin/main.lua

```diff
@@ -0,0 +1,165 @@
+local ConfirmBox = require("ui/widget/confirmbox")
+local DataStorage = require("datastorage")
+local DocSettings = require("docsettings")
+local ReadCollection = require("readcollection")
+local ReadHistory = require("readhistory")
+local ReaderUI = require("apps/reader/readerui")
+local FileManager = require("apps/filemanager/filemanager")
+local InfoMessage = require("ui/widget/infomessage")
+local LuaSettings = require("frontend/luasettings")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local logger = require("logger")
+local util = require("frontend/util")
+local BaseUtil = require("ffi/util")
+local _ = require("gettext")
+
+local MoveToArchive = WidgetContainer:new{
+    name = "move2archive",
+    move_to_archive_settings = nil,
```

Comment:
Why so verbose?

Reply:
for readablity. 
other plugins have the same thing e.g. : plugins/evernote.koplugin/main.luam plugins/perceptionexpander.koplugin/main.lua etc

Reply:
I can't tell what you're referring to in EverNote, but PerceptionExpander surely does not:

https://github.com/koreader/koreader/blob/be72203b8302730b69f92b34acecb97baa96a83a/plugins/perceptionexpander.koplugin/main.lua#L16-L28

Note how it says `settings`, not `perception_expander_settings`.

NB Other plugins do it too is not a good argument.